### PR TITLE
Add stream_id attribute support

### PR DESCRIPTION
**Pull request by le-zell**

### CODE REVIEW

File: src/main/java/com/github/jamesnetherton/zulip/client/api/message/Message.java [hunk 1]
```diff
@@ -170,4 +173,12 @@ public List<String> getTopicLinks() {
...
     }
+    
+    public Long getStreamId() {
+        return streamId;
+	}
+    
+    public void setStreamId(Long streamId) {
+		this.streamId = streamId;
+	}
 }
```
Comment (jamesnetherton) [inline src/main/java/com/github/jamesnetherton/zulip/client/api/message/Message.java:184]: Minor formatting suggestion:

```suggestion
    public Long getStreamId() {
        return streamId;
    }

    public void setStreamId(Long streamId) {
        this.streamId = streamId;
    }
```

File: src/main/java/com/github/jamesnetherton/zulip/client/api/message/Message.java [hunk 2]
```diff
@@ -58,6 +58,9 @@ public class Message {
     private String senderRealm;
 
     private String stream;
+    
+    @JsonProperty("stream_id")
```
Comment (jamesnetherton) [inline src/main/java/com/github/jamesnetherton/zulip/client/api/message/Message.java:62]: Jackson will automatically serialize the JSON field names to snake case. So you can simplify to:
```suggestion
    @JsonProperty
```

**Review by jamesnetherton — COMMENTED** [review]
Thanks!

I left a couple of comments & minor suggestions.

File: src/main/java/com/github/jamesnetherton/zulip/client/api/message/Message.java (hunk 2 above)
Comment (le-zell) [reply src/main/java/com/github/jamesnetherton/zulip/client/api/message/Message.java:62]: I wanted to user lombok @Data annotation and others. do you think it's a good idea

Comment (jamesnetherton) [reply src/main/java/com/github/jamesnetherton/zulip/client/api/message/Message.java:62]: I'm not a big fan of Lombok unfortunately.

I know it means potentially writing more boilerplate, but I don't mind taking that hit or getting my IDE / chatbots to generate code for me if needed.

**Comment by jamesnetherton** [comment]
I merged this manually as using the GitHub UI to commit the suggested changes seems to have screwed up the source branch.

Thanks for contributing @le-zell 👍

**Pull request closed without merging**